utils/preview.ts

```diff
@@ -49,14 +50,17 @@ const main = async () => {
 
   await validatePath(QUICKSTART_DIRECTORY);
 
+  app.use(cors({
+    origin: 'http://localhost:8000'
```

Comment:
We'll probably just want this to use the `port` variable as well.
```suggestion
    origin: `http://localhost:${port}`
```

Reply:
Scratch that, I'm getting the two sides turned around.
But we probably don't want to lock this down to just one URL, that way if it changes we don't need to go back and update this.

Reply:
Got it, updated to just use a wildcard. Thanks!